packages/miniflare/src/plugins/core/errors/index.ts

```diff
@@ -235,7 +235,14 @@ export function reviveError(
 	error.stack = jsonError.stack;
 
 	// Try to apply source-mapping to the stack trace
-	error.stack = getSourceMappedStack(workerSrcOpts, error);
+	error.stack = processStackTrace(
+		getSourceMappedStack(workerSrcOpts, error),
+		(line, location) =>
+			!location.includes(".wrangler/tmp") &&
+			!location.includes("wrangler/templates/middleware")
+				? line
+				: null
+	);
```

Comment:
We are now filtering out internal stack frame when we revive the error, so that Youch and error logs are consistent.